roles/archive-verify-ws/tasks/2_install_config.yml

```diff
@@ -8,65 +8,49 @@
 - name: deploy {{ arteria_service_name }} app config
   template:
     src: archive_verify_app.config.j2
-    dest: "{{ arteria_service_config_root }}/app.config"
+    dest: "{{ arteria_service_config_root }}/app.yaml"
 
 - name: deploy {{ arteria_service_name }} logger config
   template:
     src: archive_verify_logger.config.j2
-    dest: "{{ arteria_service_config_root }}/logger.config"
+    dest: "{{ arteria_service_config_root }}/logger.yaml"
 
-- name: modify uppsala's supervisord conf to set environment
+- name: modify uppsala's supervisord conf to set environment and program commands
   ini_file:
     dest: "{{ supervisord_conf }}"
-    section: supervisord
-    option: environment
-    value: LC_ALL='en_US.UTF-8',LANG='en_US.UTF-8'
+    section: "{{ item.sect }}"
+    option: "{{ item.opt }}"
+    value: "{{ item.val }}"
     backup: no
-
-- name: modify uppsala's supervisord conf to start {{ arteria_service_name }}
-  ini_file: 
-    dest: "{{ supervisord_conf }}"
-    section: program:{{ arteria_service_name }}-{{ deployment_environment }}
-    option: command
-    value: "{{ arteria_service_env_root }}/bin/pipenv run {{ arteria_service_src_path }}/archive-verify-ws --configroot={{ arteria_service_config_root }} --port={{ archive_verify_port }}"
-    backup: no
- 
-- name: modify uppsala's supervisord conf to start {{ arteria_service_name }}
-  ini_file: 
-    dest: "{{ supervisord_conf }}"
-    section: program:{{ arteria_service_name }}-{{ deployment_environment }}
-    option: autorestart
-    value: true
-    backup: no
-
-- name: modify uppsala's supervisord conf to start redis-server
-  ini_file:
-    dest: "{{ supervisord_conf }}"
-    section: program:redis-server
-    option: command
-    value: "{{ arteria_service_env_root }}/bin/redis-server"
-    backup: no
-
-- name: modify uppsala's supervisord conf to start redis-server
-  ini_file:
-    dest: "{{ supervisord_conf }}"
-    section: program:redis-server
-    option: autorestart
-    value: true
-    backup: no
-
-- name: modify uppsala's supervisord conf to start redis-worker
-  ini_file:
-    dest: "{{ supervisord_conf }}"
-    section: program:redis-worker
-    option: command
-    value: "{{ arteria_service_env_root }}/bin/pipenv run rq worker"
-    backup: no
-
-- name: modify uppsala's supervisord conf to start redis-worker
+  loop:
+    - sect: supervisord
+      opt: environment
+      val: LC_ALL='en_US.UTF-8',LANG='en_US.UTF-8'
+    - sect: program:{{ arteria_service_name }}-{{ deployment_environment }}
+      opt: command
+      val: "{{ arteria_service_env_root }}/bin/archive-verify-ws -c {{ arteria_service_config_root }}"
```

Comment:
I think it was mostly because it was redundant and potentially confusing since it's read from the config if not supplied on the command line (and it was defined in the config as well)

Reply:
Oh, I thought I had checked for it being in the config file and not finding it. But indeed, there it is. Sorry :sweat_smile: